### PR TITLE
Use matrix-doc for SAS emoji translations

### PR DESCRIPTION
**Pull request by turt2live**
**Requires https://github.com/matrix-org/matrix-js-sdk/pull/1440**

This is done through the js-sdk's new SASEmojiV1 class. See https://github.com/matrix-org/matrix-js-sdk/pull/1440

**Note**: this also prunes translations to remove the 64 strings from all languages (and apparently others we forgot about).

Proof this works:
image

### CODE REVIEW

**Comment by turt2live** [comment]
Build failing due to Synapse being broken, and lack of branch matching on some of the js-sdk PRs.

File: src/languageHandler.tsx [hunk 1]
```diff
@@ -144,6 +153,11 @@ export function _t(text: string, variables?: IVariables, tags?: Tags): string |
     }
 }
 
+export function _tSasV1(emoji: string): string | React.ReactNode {
```
Comment (t3chguy) [inline src/languageHandler.tsx:156]: this should only ever return a string as we don't pass options though

Comment (turt2live) [reply src/languageHandler.tsx:156]: Unless a developer changes the constant to have it return a node

Comment (t3chguy) [reply src/languageHandler.tsx:156]: The definition of `_t` in the usage you have says `string`

`export function _t(text: string, variables?: IVariables): string;`

Comment (turt2live) [reply src/languageHandler.tsx:156]: hmm, alright. The types seem incorrect but trying to thread the code path through my head isn't working either.

**Pull request merged**